ee/src/main/java/org/jboss/as/ee/subsystem/ManagedExecutorServiceAdd.java

```diff
@@ -104,26 +104,26 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
 
         final AbstractManagedExecutorService.RejectPolicy rejectPolicy = AbstractManagedExecutorService.RejectPolicy.valueOf(ManagedExecutorServiceResourceDefinition.REJECT_POLICY_AD.resolveModelAttribute(context, model).asString());
 
+        final ServiceName serviceName = ManagedExecutorServiceResourceDefinition.CAPABILITY.getCapabilityServiceName(name);
         final ManagedExecutorServiceService service = new ManagedExecutorServiceService(name, jndiName, hungTaskThreshold, longRunningTasks, coreThreads, maxThreads, keepAliveTime, keepAliveTimeUnit, threadLifeTime, queueLength, rejectPolicy);
-        final ServiceBuilder<ManagedExecutorServiceAdapter> serviceBuilder = context.getServiceTarget().addService(ConcurrentServiceNames.getManagedExecutorServiceServiceName(name), service);
+        final ServiceBuilder<ManagedExecutorServiceAdapter> serviceBuilder = context.getServiceTarget().addService(serviceName, service);
```

Comment:
Use context.getCapabilityServiceTarget().addCapability(...) if possible.

Reply:
seems only possible for not dynamic caps

Reply:
done